### ISSUE TITLE
Problem: Task Template ARGS Tags Not Working

### ISSUE THREAD

**Issue by rbh1981** [ansible-semaphore/semaphore#2258]
### Issue

Using --tags in 'Args' section in Task Templates causes job to fail

### Impact

Web-Frontend (what users interact with), Ansible (task execution)

### Installation method

Package

### Database

BoltDB

### Browser

Firefox

### Semaphore Version

2.10.22-e44910d-1721658299

### Logs & errors

![Screenshot from 2024-08-04 08-24-52](https://github.com/user-attachments/assets/59e985c7-ea80-4418-8254-4e797c7e6914)

### Manual installation - system information

nginx

### Configuration

![Screenshot from 2024-08-04 08-25-31](https://github.com/user-attachments/assets/90a3915d-51f3-4359-96c7-68ffed42f896)

### Additional information

Adding the arg --check to the task template seems to work but --tags does not. Running the playbook from the commands line with --tags works as expected.

**Comment by MascHman** [comment]
Hey,

it's working for me with the Arg "-t users".

**Comment by m-vandeneede** [comment]
Same issue here, using -t works but I would also like to use --skip-tags

**Comment by rbh1981** [comment]
Agree. The old style of specifying additional args as json array seemed to work without issue. The new UI for specifying additional args seems limited.

**Comment by fiftin** [comment]
Hi all, thank you! Fix be fixed soon.

**Comment by fiftin** [comment]
What if try `--tags=users`?  Or you should use 2 items `--tags`, `users`.

**Comment by rbh1981** [comment]
> What if try `--tags=users`? Or you should use 2 items `--tags`, `users`.

Thanks for the tip. Separating args for '--tags' and tag name does work as expected. I guess it is just a misunderstanding on how args need to be used in the UI? 

Screenshot shows working tags as recommended:

![Screenshot from 2024-08-21 16-46-16](https://github.com/user-attachments/assets/b9c78413-4efc-40b1-9484-306d33798062)

**Comment by powerpaul17** [comment]
I can confirm that `--tags=users` also works.

**Comment by richard-foster** [comment]
Hi @fiftin, do you have an ETA for fixing this? Once fixed, it'll unlock wonderful things for us... at the moment we have to duplicate dozens of playbooks...! Thanks

**Comment by derdems** [comment]
I can confirm that --skip-tags and <tagname> as individual entries just works as if executed in CLI.
This probably needs to be communicated more exactly.
